pyglet/image/atlas.py

```diff
@@ -252,7 +252,7 @@ class TextureArrayBin:
     ones as necessary as the depth is exceeded.
     """
 
-    def __init__(self, texture_width=2048, texture_height=2048, max_depth=None):
+    def __init__(self, texture_width: int=2048, texture_height: int=2048, max_depth: Optional[int]=None):
```

Comment:
how about `texture_width: int = 2048` ?

Reply:
It should be without spaces

Reply:
> It should be without spaces

well, my pycharm say's it should have spaces?